hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -108,3 +108,13 @@ source $CDC_WAIVER_FILE
 #########################
 
 report_policy -verbose -skip_empty_summary_status -compat -output vcdc.rpt ALL
+
+set modules {spi_device kmac hmac uart gpio spi_host flash_ctrl alert_handler otp_ctrl lc_ctrl pwrmgr clkmgr rstmgr keymgr csrng entropy_src aes}
```

Comment:
That is great! We should probably enhance our dashboard at some point to parse the reports out correctly.

Reply:
One issue is that the tool changes the module names after elaboration if the module is instantiated multiple times (e.g. uart/ spi_host). Need to resolve this.